### ISSUE TITLE
Separate Database Name and Attribute Name

### ISSUE THREAD

**Issue by kadenbarlow** [ankane/lockbox#67]
New to lockbox but loving it so far!

This may be possible but after reading through the readme and code I couldn't see that it was. 
Currently, in attribute encrypted, I have something like

```
  attr_encrypted :email_address, key: ..., attribute: 'encrypted_email'
```
I'm able to specify a specific database column that has the encrypted data instead of using the default column name conventions. Is there any way to do this with Lockbox?

**Comment by ankane** [comment]
Hey @kadenbarlow, added an `encrypted_attribute` option on master. The name may change before the next release (I'm considering `ciphertext_attribute` as well), but will try to update here if it does.

**Comment by ankane** [comment]
Released in 0.4.9.
